plutus-wallet-api/ledger/Ledger/Slot.hs

```diff
@@ -41,7 +40,7 @@ import           Ledger.Interval
 -- slots pass at a constant rate.
 newtype Slot = Slot { getSlot :: Integer }
     deriving stock (Haskell.Eq, Haskell.Ord, Show, Generic)
-    deriving anyclass (ToSchema, FromJSON, FromJSONKey, ToJSON, ToJSONKey, IotsType)
+    deriving anyclass ( FromJSON, FromJSONKey, ToJSON, ToJSONKey, IotsType)
```

Comment:
Given we're dropping all these, can we also drop the dependency of `plutus-wallet-api:ledger` on `playground-common`?

Reply:
Yup, good spot. (Has anyone ever got weeder to work by the way?)